src/app/ReviewService/ReviewPrompter.Windows.cs

```diff
@@ -0,0 +1,28 @@
+﻿#if WINDOWS
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.UI.Dispatching;
+using ReviewService.Abstractions;
+
+namespace ReviewService;
+
+/// <summary>
+/// Windows implementation of <see cref="IReviewPrompter"/>.
+/// </summary>
+public sealed class ReviewPrompter : IReviewPrompter
+{
+	private readonly DispatcherQueue _dispatcherQueue;
+
+	public ReviewPrompter(DispatcherQueue dispatcherQueue)
```

Comment:
Do we even need a DispatcherQueue on windows? Is there native support for rating on WinUI?

Reply:
Package said it support Windows, not sure if it means WinUI. https://github.com/jamesmontemagno/StoreReviewPlugin

Reply:
Just does nothing! Will remove it.